gear-test/src/manager.rs

```diff
@@ -93,6 +93,11 @@ impl JournalHandler for InMemoryExtManager {
             }
         };
     }
+
+    fn exit_dispatch(&mut self, _id_exited: ProgramId, _value_destination: ProgramId) {
+        unimplemented!()
```

Comment:
And this is the second place I'm uncertain with. I'm able to implement `exit_dispatch` in every `JournalHandler` though it would not be called at the moment. So I decided to go with `umiplmemented`.